### ISSUE TITLE
Cannot upgrade from 4.2.0 to 4.3.0

### ISSUE THREAD

**Issue by Sonic3R** [thelounge/thelounge#4374]
- _Node version:_ v16.13.0
- _Browser version:_ any
- _Device, operating system:_ Ubuntu 20.04.3 LTS
- _The Lounge version:_ actual 4.2.0

---

I tried to run apt get install ./thelounge.deb and I get (after I chose to preserve current config.js file) :

```
Downloading sqlite3 module for your specific installation
yarn install v1.22.10
[1/5] Validating package.json...
[2/5] Resolving packages...
error Couldn't find the binary git
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
[!!] Failed to install sqlite3 module correctly, The Lounge will continue working, but you might want to fix this.
Removing leftover files
Warning: The unit file, source configuration file or drop-ins of thelounge.service changed on disk. Run 'systemctl daemon-reload' to reload units.
```

I used `systemctl daemon-reload` and then check thelounge process status and it said that is fail to start

```
 $ systemctl status thelounge

● thelounge.service - The Lounge (IRC client)
     Loaded: loaded (/lib/systemd/system/thelounge.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Thu 2021-11-25 08:58:59 CET; 3s ago
    Process: 414684 ExecStart=/usr/bin/thelounge start (code=exited, status=1/FAILURE)
   Main PID: 414684 (code=exited, status=1/FAILURE)

Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Array.forEach (<anonymous>)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Array.forEach (<anonymous>)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
```

Before upgrading, I stopped the service with `systemctl stop thelounge`

But rollback to 4.2.0 worked and thelounge is running. But not for 4.3.0

**Comment by brunnre8** [comment]
Please give the full log leading to the error, the important bits are cut off

**Comment by Sonic3R** [comment]
How ? I pasted from what I got in terminal

**Comment by brunnre8** [comment]
Sure, but that's not the command you should be using for logs.
systemctl status simply adds a few lines of log output for convenience.

To get the logs, `journalctl -u thelounge.service`
Then find the appropriate section, the above gives you everything since the beginning of time.
You could restrict it with flags, personally I simply scroll to the end and back up to the thing I'm looking for.

**Comment by brunnre8** [comment]
If that isn't working because Ubuntu pulls a Debian and doesn't have a persistent journal they probably shove it into /var/log/$something so check there.

(just to avoid a third roundtrip)

**Comment by Sonic3R** [comment]
Logs from journal:

```
Nov 23 14:06:04 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 23 14:06:04 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
Nov 23 14:10:31 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]: 2021-11-23 13:10:32 [INFO] The Lounge v4.3.0 (Node.js 16.13.0 on linux x64)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]: 2021-11-23 13:10:32 [INFO] Configuration file: /etc/thelounge/config.js
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]: 2021-11-23 13:10:32 [INFO] Available at http://[::]:9000/ in private mode
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]: /usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:             type: socket_io_parser_1.PacketType.EVENT,
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:                                                 ^
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]: TypeError: Cannot read properties of undefined (reading 'EVENT')
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at Namespace.emit (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170:49)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at Server.<computed> [as emit] (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/index.js:466:33)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at Array.forEach (<anonymous>)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal thelounge[2150283]:     at Array.forEach (<anonymous>)
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 23 14:10:32 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
Nov 23 14:12:22 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]: 2021-11-23 13:12:23 [INFO] The Lounge v4.3.0 (Node.js 16.13.0 on linux x64)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]: 2021-11-23 13:12:23 [INFO] Configuration file: /etc/thelounge/config.js
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]: 2021-11-23 13:12:23 [INFO] Available at http://[::]:9000/ in private mode
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]: /usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:             type: socket_io_parser_1.PacketType.EVENT,
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:                                                 ^
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]: TypeError: Cannot read properties of undefined (reading 'EVENT')
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at Namespace.emit (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170:49)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at Server.<computed> [as emit] (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/index.js:466:33)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at Array.forEach (<anonymous>)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal thelounge[2150590]:     at Array.forEach (<anonymous>)
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 23 14:12:23 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
Nov 23 14:17:02 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]: 2021-11-23 13:17:03 [INFO] The Lounge v4.3.0 (Node.js 16.13.0 on linux x64)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]: 2021-11-23 13:17:03 [INFO] Configuration file: /etc/thelounge/config.js
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]: 2021-11-23 13:17:03 [INFO] Available at http://[::]:9000/ in private mode
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]: /usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:             type: socket_io_parser_1.PacketType.EVENT,
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:                                                 ^
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]: TypeError: Cannot read properties of undefined (reading 'EVENT')
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at Namespace.emit (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170:49)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at Server.<computed> [as emit] (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/index.js:466:33)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at Array.forEach (<anonymous>)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal thelounge[2151356]:     at Array.forEach (<anonymous>)
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 23 14:17:03 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
-- Reboot --
Nov 23 14:20:34 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]: 2021-11-23 13:20:41 [INFO] The Lounge v4.3.0 (Node.js 16.13.0 on linux x64)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]: 2021-11-23 13:20:41 [INFO] Configuration file: /etc/thelounge/config.js
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]: 2021-11-23 13:20:41 [INFO] Available at http://[::]:9000/ in private mode
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]: /usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:             type: socket_io_parser_1.PacketType.EVENT,
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:                                                 ^
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]: TypeError: Cannot read properties of undefined (reading 'EVENT')
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at Namespace.emit (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170:49)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at Server.<computed> [as emit] (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/index.js:466:33)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at Array.forEach (<anonymous>)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal thelounge[447]:     at Array.forEach (<anonymous>)
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 23 14:20:41 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
Nov 23 14:24:57 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 23 14:24:59 Ubuntu-2004-focal-64-minimal thelounge[1512]: 2021-11-23 13:24:59 [INFO] The Lounge v4.2.0 (Node.js 16.13.0 on linux x64)
Nov 23 14:24:59 Ubuntu-2004-focal-64-minimal thelounge[1512]: 2021-11-23 13:24:59 [INFO] Configuration file: /etc/thelounge/config.js
Nov 23 14:24:59 Ubuntu-2004-focal-64-minimal thelounge[1512]: 2021-11-23 13:24:59 [INFO] Available at http://[::]:9000/ in private mode
Nov 23 14:24:59 Ubuntu-2004-focal-64-minimal thelounge[1512]: 2021-11-23 13:24:59 [INFO] User sonic3r loaded
Nov 23 14:25:00 Ubuntu-2004-focal-64-minimal thelounge[1512]: 2021-11-23 13:25:00 [INFO] The Lounge v4.3.0 is available. Read more on GitHub: https://github.com/thelounge/thelounge/releases/tag/v4.3.0
Nov 25 08:57:30 Ubuntu-2004-focal-64-minimal thelounge[1512]: 2021-11-25 07:57:29 [INFO] Exiting...
Nov 25 08:57:30 Ubuntu-2004-focal-64-minimal systemd[1]: Stopping The Lounge (IRC client)...
Nov 25 08:57:30 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Succeeded.
Nov 25 08:57:30 Ubuntu-2004-focal-64-minimal systemd[1]: Stopped The Lounge (IRC client).
Nov 25 08:58:14 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]: 2021-11-25 07:58:16 [INFO] The Lounge v4.3.0 (Node.js 16.13.0 on linux x64)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]: 2021-11-25 07:58:16 [INFO] Configuration file: /etc/thelounge/config.js
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]: 2021-11-25 07:58:16 [INFO] Available at http://[::]:9000/ in private mode
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]: /usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:             type: socket_io_parser_1.PacketType.EVENT,
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:                                                 ^
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]: TypeError: Cannot read properties of undefined (reading 'EVENT')
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at Namespace.emit (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170:49)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at Server.<computed> [as emit] (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/index.js:466:33)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at Array.forEach (<anonymous>)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal thelounge[414416]:     at Array.forEach (<anonymous>)
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 25 08:58:16 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
Nov 25 08:58:58 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]: 2021-11-25 07:58:59 [INFO] The Lounge v4.3.0 (Node.js 16.13.0 on linux x64)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]: 2021-11-25 07:58:59 [INFO] Configuration file: /etc/thelounge/config.js
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]: 2021-11-25 07:58:59 [INFO] Available at http://[::]:9000/ in private mode
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]: /usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:             type: socket_io_parser_1.PacketType.EVENT,
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:                                                 ^
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]: TypeError: Cannot read properties of undefined (reading 'EVENT')
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Namespace.emit (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170:49)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Server.<computed> [as emit] (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/index.js:466:33)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Array.forEach (<anonymous>)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal thelounge[414684]:     at Array.forEach (<anonymous>)
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Main process exited, code=exited, status=1/FAILURE
Nov 25 08:58:59 Ubuntu-2004-focal-64-minimal systemd[1]: thelounge.service: Failed with result 'exit-code'.
Nov 25 09:00:10 Ubuntu-2004-focal-64-minimal systemd[1]: Started The Lounge (IRC client).
Nov 25 09:00:11 Ubuntu-2004-focal-64-minimal thelounge[414986]: 2021-11-25 08:00:11 [ERROR] Unable to load node-sqlite3 module. See https://github.com/mapbox/node-sqlite3/wiki/Binaries
Nov 25 09:00:12 Ubuntu-2004-focal-64-minimal thelounge[414986]: 2021-11-25 08:00:12 [INFO] The Lounge v4.2.0 (Node.js 16.13.0 on linux x64)
Nov 25 09:00:12 Ubuntu-2004-focal-64-minimal thelounge[414986]: 2021-11-25 08:00:12 [INFO] Configuration file: /etc/thelounge/config.js
Nov 25 09:00:12 Ubuntu-2004-focal-64-minimal thelounge[414986]: 2021-11-25 08:00:12 [INFO] Available at http://[::]:9000/ in private mode
Nov 25 09:00:13 Ubuntu-2004-focal-64-minimal thelounge[414986]: 2021-11-25 08:00:13 [INFO] User sonic3r loaded
Nov 25 09:00:13 Ubuntu-2004-focal-64-minimal thelounge[414986]: 2021-11-25 08:00:13 [INFO] The Lounge v4.3.0 is available. Read more on GitHub: https://github.com/thelounge/thelounge/releases/tag/v4.3.0

```

**Comment by SirLouen** [comment]
I have found the exact same issue here

My system: 
Ubuntu 20.04.3

Upgrading through [azlux](http://packages.azlux.fr/debian) bullseye repo

```
Feb 02 17:15:27 thelounge thelounge[532]: 2022-02-02 16:15:27 [INFO] The Lounge v4.3.0 (Node.js 16.13.2 on linux x64)
Feb 02 17:15:27 thelounge thelounge[532]: 2022-02-02 16:15:27 [INFO] Configuration file: /etc/thelounge/config.js
Feb 02 17:15:27 thelounge thelounge[532]: 2022-02-02 16:15:27 [INFO] Available at http://127.0.0.1:9000/ in private mode
Feb 02 17:15:28 thelounge thelounge[532]: /usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170
Feb 02 17:15:28 thelounge thelounge[532]:             type: socket_io_parser_1.PacketType.EVENT,
Feb 02 17:15:28 thelounge thelounge[532]:                                                 ^
Feb 02 17:15:28 thelounge thelounge[532]: TypeError: Cannot read properties of undefined (reading 'EVENT')
Feb 02 17:15:28 thelounge thelounge[532]:     at Namespace.emit (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/namespace.js:170:49)
Feb 02 17:15:28 thelounge thelounge[532]:     at Server.<computed> [as emit] (/usr/lib/node_modules/thelounge/node_modules/socket.io/dist/index.js:466:33)
Feb 02 17:15:28 thelounge thelounge[532]:     at Client.emit (/usr/lib/node_modules/thelounge/src/client.js:158:36)
Feb 02 17:15:28 thelounge thelounge[532]:     at Client.connect (/usr/lib/node_modules/thelounge/src/client.js:267:9)
Feb 02 17:15:28 thelounge thelounge[532]:     at /usr/lib/node_modules/thelounge/src/client.js:121:61
Feb 02 17:15:28 thelounge thelounge[532]:     at Array.forEach (<anonymous>)
Feb 02 17:15:28 thelounge thelounge[532]:     at new Client (/usr/lib/node_modules/thelounge/src/client.js:121:33)
Feb 02 17:15:28 thelounge thelounge[532]:     at ClientManager.loadUser (/usr/lib/node_modules/thelounge/src/clientManager.js:140:12)
Feb 02 17:15:28 thelounge thelounge[532]:     at /usr/lib/node_modules/thelounge/src/clientManager.js:78:32
Feb 02 17:15:28 thelounge thelounge[532]:     at Array.forEach (<anonymous>) ```

Downgraded back to 4.2.0 and everything worked flawlessly.

Not sure what's going on, but is clearly an issue with something upgraded on 4.3.0

**Comment by xPaw** [comment]
What it sounds like is `node_modules` was not properly updated during the upgrade. Are you still getting this with the latest release? Does this only happen with our provided .deb?
